packages/lumx-react/src/components/mosaic/__snapshots__/Mosaic.test.tsx.snap

```diff
@@ -16,7 +16,6 @@ exports[`<Mosaic> Snapshots and structure should render story fourThumbnails 1`]
         fillHeight={true}
         image="https://picsum.photos/200"
         onClick={[Function]}
-        tabIndex="0"
```

Comment:
Should you provide the "thumbnail.onClick" function instead of updating the snap?

Reply:
I'm not sure I follow ? 
Should we have stories with and without `onClick` for each cases ? (1 / 2 /3 / 4 / 5+ thumbnails)

Reply:
@radiuso  I have added specific tests for the tabIndex prop => https://github.com/lumapps/design-system/pull/398/files#diff-74838aefe74fc3c326a7d3d54cbd4a0e

Reply:
Also added a story for "Clickable thumbnails".